MathlibTest/FieldSimp.lean

```diff
@@ -516,18 +554,6 @@ example {V : Type*} [AddCommGroup V] (F : V → ℚ)
     = -((F x * F x + F y * F y - F (x - y) * F (x - y)) / 2 / (F x * F y))
         * F x * F y * F (x - y) * F (x - y) := by
   field_simp
-  guard_target =
-    ((F x * F x * 2 - (F x * F x + F y * F y - F (x - y) * F (x - y)))
-      * (F y * F y * 2 - (F x * F x + F y * F y - F (x - y) * F (x - y)))
-      * F x * F y * F (x - y) * F (x - y) * (2 * 2)
-      - 2 * (F x * F (x - y)) * (2 * (F y * F (x - y))) *
-        (F x * F x * (F y * F y) * (2 * 2)
-        - (F x * F x + F y * F y - F (x - y) * F (x - y))
-        * (F x * F x + F y * F y - F (x - y) * F (x - y))))
-    * (2 * (F x * F y))
-    = (F (x - y) * F (x - y) - (F x * F x + F y * F y))
-      * F x * F y * F (x - y) * F (x - y)
-      * (2 * (F x * F (x - y)) * (2 * (F y * F (x - y))) * (2 * 2))
```

Comment:
Deleting this check because it is time-consuming enough to seriously affect the heartbeat count!  (In the new implementation it accounts for 2/3 of the heartbeats.)